src/test/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/model/ActionConfigTests.kt

```diff
@@ -56,4 +57,12 @@ class ActionConfigTests : ESTestCase() {
             randomRolloverActionConfig(minDocs = 0)
         }
     }
+
+    fun `test force merge action max num segments of zero fails`() {
+        assertFailsWith(
```

Comment:
np: Was this over the 140 limit we currently have? this seems harder to read than
```
assertFailsWith(IllegalArgumentException::class, "Expected IllegalArgumentException for maxNumSegments less than 1") {
    randomForceMergeActionConfig(maxNumSegments = 0)
}

```

Reply:
These tests were written before the change to 140 character line limit. Changed the format to make it easier to read.